### ISSUE TITLE
Segfault

### ISSUE THREAD

**Issue by Duality4Y** [CTPUG/pygame_cffi#95]
Hi!
So have been using pygame_cffi for a while and one day noticed that one of my programs using pygame_cffi would segfault once in a while.
I figured out what causes it and how to reproduce:
```python
import pygame
import time

pygame.init()
surface = pygame.display.set_mode((800, 800))

color = [255, 127, 0]
pos = (656, 800)
radius = 0
width = 0

pygame.draw.circle(surface, color, pos, radius, width)

```

objects in my program would get smaller with distance sometimes their radius would reach zero, not often though. so initially i did not pick it up.

if width > radius it throws width greater then radius.
and if radius > width it would work fine.
so it is only in the case of radius = width = 0
that it would segfault.

for me it's an easy fix i can live with
```python
    if radius == 0:
        radius = 1
```

but it shouldn't segfault.

**Comment by drnlm** [comment]
Thanks for the bug report and the test case. It made tracking down the issue very easy.

Fixed with PR #96

**Comment by Duality4Y** [comment]
oh nice and quick! :)
